river/rule/amrules.py

```diff
@@ -95,6 +96,25 @@ def drift_test(self, y, y_pred):
         return in_drift
 
     def anomaly_score(self, x):
+        """Rule anomaly score.
+
+        The more negative the score, the more anomalous is the instance regarding the subspace
+        covered by the rule. Instances whose score is greater than zero are considered normal.
+        Different threshold values can be used to decide upon discarding or using the instance
+        for training. A small negative threshold is considered to be a conservative choice,
+        whereas cutting at zero is a more aggressive choice.
```

Comment:
If you inherit from `AnomalyDetector`, you can just use the description from there to explain this method, no need to repeat ourselves. Your explanation is more verbose so don't hesitate to merge both comments into something better.

Reply:
Thanks for the tip! I did as you said. I kept most from the old description because the Rules' scores have a different interpretation than the base anomaly detector. Negative values indicate anomalies, whereas the positive ones indicate normal examples.